src/kirby/components/avatar/avatar.component.tns.html

```diff
@@ -14,7 +14,8 @@
     horizontalAlignment="center" 
     verticalAlignment="center" 
     [visibility]="imageSrc ? 'collapse' : 'visible'">
-    <ng-content select="kirby-icon"></ng-content>
+    <ng-content *ngIf="!text" select="kirby-icon"></ng-content>
+    <Label *ngIf="text" class="avatar-text" horizontalAlignment="center" text="{{text}}" verticalAlignment="center"></Label>
```

Comment:
With these checks, would the user not be able to have an `kirby-avatar` with both `imageSrc` and `text`?

Reply:
No, if the imageSrc is present, this would be collapsed.

Reply:
Would i not be able to do 

```html
<kirby-avatar [imageSrc]="src" text="J"></kirby-avatar>
```

i don't see any checks for hidden either one of image or text, if both are selected. Can be we just expect the user to be clever about it. 😄 

One more thing (nitpicking):

```suggestion
    <Label *ngIf="text" class="avatar-text" horizontalAlignment="center" [text]="text" verticalAlignment="center"></Label>
```

I personally find it easier to read, when not using string interpolation inside attributes where binding is possible. 😄